src/client/components/Input/Input.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Input.css';
+
+export default function Input(props) {
+  const [value, setValue] = useState();
+
+  const onChange = (event) => {
+    setValue(event.target.value);
+  };
+  return (
+    <input
+      className="input-component"
+      value={value}
+      onChange={onChange}
+      placeholder={props.text}
```

Comment:
You should rename `text` to `placeholder` and add it to `propTypes`.
Also, `value` and `onChange` should be passed from `props` too.